test/mainnet/helpers/SetupWithdrawer.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: BSD Clause-3
+pragma solidity ^0.8.24;
+
+import {Test} from "lib/forge-std/src/Test.sol";
+import {Withdrawer} from "src/withdraws/Withdrawer.sol";
+import {TransparentUpgradeableProxy as TUProxy} from "src/Common.sol";
+import {Etches} from "test/unit/helpers/Etches.sol";
+import {MainnetActors} from "script/Actors.sol";
+import {MainnetContracts as MC} from "script/Contracts.sol";
+import {WithdrawerUtils} from "script/WithdrawerUtils.sol";
+
+contract SetupWithdrawer is Test, Etches, MainnetActors, WithdrawerUtils {
+    function setup() public returns (Withdrawer vault) {
+        string memory name = "YieldNest Withdrawer";
+        string memory symbol = "ynWithdrawer";
+
+        Withdrawer vaultImplementation = new Withdrawer();
+
+        // Deploy the proxy
+        bytes memory initData =
+            abi.encodeWithSelector(Withdrawer.initialize.selector, ADMIN, name, symbol, 18, true, true);
```

Comment:
set alwaysComputeTotalAssets_ to false, that's how we want to use this in production

Reply:
do this for SetupWithdrawer.sol in the unit test directory